sh.shuriken.Eeman.json

```diff
@@ -91,7 +91,7 @@
                     "type" : "git",
                     "url" : "https://codeberg.org/SHuRiKeN/Eeman",
                     "tag" : "0.1.0",
-                    "commit" : "26231f1617039b264bf068dc2104e1163b10b5dc"
+                    "commit" : "647d48da68de328a876e6551de3f61441663809f"
```

Comment:
retagging is a very bad practice.

Reply:
I thought it won't matter before the builds land..

Reply:
it has nothing to do with flathub. It is in general a bad practice regardless.

Reply:
Hmm.. will simply increment tags in the future.